js/base/Exchange.js

```diff
@@ -859,6 +859,50 @@ module.exports = class Exchange {
         };
     }
 
+    currencyStructure () {
+        return {
+            'info': undefined,
+            'id': undefined,
+            'lowercaseId': undefined,
+            'uppercaseId': undefined,
+            'numericId': undefined,
+            'code': undefined,
+            'precision': undefined,
+            'type': undefined,
+            'margin': undefined,
+            'name': undefined,
+            'active': undefined,
+            'deposit': undefined,
+            'withdraw': undefined,
+            'fee': undefined,
+            'fees': undefined,
+            'networks': undefined,
+            'address': undefined,
```

Comment:
what is this field for? inside `networks` we have different address, and the "generic" address for coin might not be correct.

Reply:
I just included all the fields that were present in the currency structure for one of the exchanges

Reply:
I think it would be mistake to include `address` field anywhere , even in one exchanges, because `address` is specifically a sub-value of network, and not a coin globally. I would vote for removal of that field.